scripts/exregional_get_extrn_mdl_files.sh

```diff
@@ -171,6 +177,49 @@ ${cmd}
 #
 #-----------------------------------------------------------------------
 #
+# Merge GEFS files
+#
+#-----------------------------------------------------------------------
+#
+if [ "${EXTRN_MDL_NAME}" = "GEFS" ]; then
+    for num in $(seq -f "%02g" ${NUM_ENS_MEMBERS}); do
+        if [ "${ICS_OR_LBCS}" = "LBCS" ]; then
+            fcst_hrs_tmp=( $fcst_hrs )
+            fcst_hrs_ary=( $(seq ${fcst_hrs_tmp[0]} ${fcst_hrs_tmp[2]} ${fcst_hrs_tmp[1]}) )
+        else
+            fcst_hrs_ary=( ${fcst_hrs} )
+        fi
+            for fcst_hr in "${fcst_hrs_ary[@]}"; do
+                fn_list_1=( "gep$num.t${hh}z.pgrb2a.0p50.f`printf %03d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2b.0p50.f`printf %03d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2.0p50.f`printf %03d $fcst_hr`" )
+                fn_list_2=( "gep$num.t${hh}z.pgrb2af`printf %02d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2bf`printf %02d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2`printf %02d $fcst_hr`" )
+                fn_list_3=( "gep$num.t${hh}z.pgrb2af`printf %03d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2bf`printf %03d $fcst_hr`" \
+                        "gep$num.t${hh}z.pgrb2`printf %03d $fcst_hr`" )
+                fn_lists=( "fn_list_1" "fn_list_2" "fn_list_3" )
```

Comment:
It does make sense to read the info in from the data locations file, but I couldn't come up with a way I liked to read them in. What I landed on was to use the grep command and grab the data locations that way. I'll be updating this file soon so you can see what I mean.

Reply:
@christinaholtNOAA - Where you wanting to read the file names from the summary file? Because I thought about doing it that way too but ultimately decided to read from the source (data_locations.yaml) instead.